cli/samplereports/README.md

```diff
@@ -0,0 +1,4 @@
+This is the folder for report queries.
```

Comment:
I think we can put this at the root (ie `github.com/GoogleCloudPlatform/cloud-foundation-toolkit/reports/sample`).

Reply:
done in https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/pull/315/commits/ebdf1fbc6190f26175d669b6b90d106f8c5bcd5f